From deeedd1b8799294ab276ab7dbbfdb59c1dacc9a2 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Thu, 22 Oct 2020 13:10:34 +0200
Subject: [PATCH] Lib/sysconfig.py: use libdir values from configuration file

This allows correctly substituting them for target installs using
native python.

Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 Lib/sysconfig.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Lib/sysconfig.py b/Lib/sysconfig.py
index bf04ac5..ed0462b 100644
--- a/Lib/sysconfig.py
+++ b/Lib/sysconfig.py
@@ -20,10 +20,10 @@ __all__ = [
 
 _INSTALL_SCHEMES = {
     'posix_prefix': {
-        'stdlib': '{installed_base}/{platlibdir}/python{py_version_short}',
-        'platstdlib': '{platbase}/{platlibdir}/python{py_version_short}',
-        'purelib': '{base}/lib/python{py_version_short}/site-packages',
-        'platlib': '{platbase}/{platlibdir}/python{py_version_short}/site-packages',
+        'stdlib': '{LIBDEST}',
+        'platstdlib': '{LIBDEST}',
+        'purelib': '{LIBDEST}/site-packages',
+        'platlib': '{LIBDEST}/site-packages',
         'include':
             '{installed_base}/include/python{py_version_short}{abiflags}',
         'platinclude':
-- 
2.24.0

